roborazzi/src/main/java/com/github/takahirom/roborazzi/Roborazzi.kt

```diff
@@ -72,6 +82,89 @@ fun ViewInteraction.captureRoboImage(
   })
 }
 
+fun View.captureRoboImage(
+  filePath: String = DefaultFileNameGenerator.generateFilePath("png"),
+  roborazziOptions: RoborazziOptions = RoborazziOptions(),
+) {
+  if (!roborazziEnabled()) return
+  captureRoboImage(
+    file = File(filePath),
+    roborazziOptions = roborazziOptions
+  )
+}
+
+fun View.captureRoboImage(
```

Comment:
I've tested this and unfortunately, it looks like I was right: it would not work for a Dialog since the view is not attached to the activity window.

However, since Bitmaps would be supported too, one could call `dialog.window.decorView.drawToBitmap().captureRoboImage()` and should be fine I believe... (could not test it)

Another option would be to get it via Espresso... but I could not find a way...

I'd prefer a solution without Espresso, but could serve the purpose for a first iteration?

Reply:
I attempted and implemented a solution, but I couldn't resolve an issue that arises when displaying a dialog stating, "java.lang.IllegalStateException: The specified child already has a parent. You must call removeView() on the child's parent first." This occurs even when I remove the view from the parent.

https://github.com/takahirom/roborazzi/pull/42/commits/5b7e51851ce514848a80143929d000f5cb60fa4a#diff-1bb3d1a101c61df64c19694d660a430f6802d62054f0963f804d9edca69fb971R86

Regardless, I believe it might be better than having no implementation, so I would like to introduce them.